src/components/Modal/Modal.css

```diff
@@ -0,0 +1,95 @@
+
+.modal__wrapper {
+  font-family: Aktiv Grotesk, -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif;
+  position: absolute;
+  z-index: 1000;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+}
+
+.modal__wrapper.--hidden {
```

Comment:
`--hidden` is not a separate class, you don't apply the class like this: `modal__wrapper --hidden`, the --hidden is directly connected to the rest of the class declaration, so you write it as `.modal__wrapper--hidden` (drop the point), and use it as `modal__wrapper--hidden` (drop the space).